plugins/device/init.go

```diff
@@ -32,12 +32,22 @@ func (p *Plugin) RestRoutes() route.Routes {
 			Version:      1,
 			RequestType:  utils.GetTypeString((*deviceapi.AddDeviceReq)(nil)),
 			ResponseType: utils.GetTypeString((*deviceapi.AddDeviceResp)(nil)),
-			HandlerFunc:  deviceAddHandler},
+			HandlerFunc:  deviceAddHandler,
+		},
+		route.Route{
+			Name:         "EditGroup",
+			Method:       "POST",
+			Pattern:      "/peers/{peerid}/group/{group_id}",
```

Comment:
We've never done this in other APIs - the request data being part of the URL. To maintain uniformity, I'd prefer if this was in the body.